test/AutoRemediateS3-008.test.js

```diff
@@ -248,37 +223,40 @@ describe('S3-008 AutoRemediation', () => {
           Owner: expect.any(Object)
         }
       }
-      expect(putBucketAclMock).toHaveBeenCalledWith(expect.objectContaining(expectedGrant), expect.any(Function))
+      expect(mockS3).toHaveReceivedCommandWith(PutBucketAclCommand, expectedGrant)
     })
   })
 
   describe('invalid invocation', () => {
-    const mockCallback = (done) => {
-      return (err, data) => {
-        expect(err).toBeDefined()
-        expect(getBucketAclMock).not.toHaveBeenCalled()
-        expect(putBucketAclMock).not.toHaveBeenCalled()
-        done()
-      }
-    }
-
-    it('should fail when event is undefined', done => {
-      source.handler(undefined, jest.fn(), mockCallback(done))
+    it('should fail when event is undefined', async () => {
+      await expect(source.handler(undefined, jest.fn()))
```

Comment:
♻️ no second argument needed
```suggestion
      await expect(source.handler(undefined))
```